plugin.json

```diff
@@ -4,8 +4,10 @@
     "description": "Zoom audio and video conferencing plugin for Mattermost.",
     "homepage_url": "https://github.com/mattermost/mattermost-plugin-zoom",
     "support_url": "https://github.com/mattermost/mattermost-plugin-zoom/issues",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.7.1",
     "icon_path": "assets/profile.svg",
     "min_server_version": "5.37.0",
+    "version": "1.7.1",
```

Comment:
Assuming we are using the automated version system now, maybe we should just create an issue for this instead of a PR. Then we can create an RC build and deploy to community. The issue can serve as our public communication about the release. If there are any additional changes to be made, we can open a PR for those changes